### PR TITLE
Remove TNF_OC_DEBUG_IMAGE_ID env var

### PR DESCRIPTION
**Pull request by javierpena**
We no longer need to specify the TNF_OC_DEBUG_IMAGE_ID environment
variable, since we are using the new debug pod image. This commit
removes all usage of the env var, and updates documentation to reflect
the current situation.

### CODE REVIEW

**Review by sebrandon1 — APPROVED** [review]

File: pkg/tnf/handlers/common/command.go [hunk 1]
```diff
@@ -2,29 +2,19 @@ package common
...
-var OcDebugImageID string
-
 const (
 	ocCommand    = "oc"
 	debugCommand = "debug"
-	imageArg     = "--image"
 )
 
 // GetOcDebugCommand returns the command base for any test handler that uses the oc debug command
 func GetOcDebugCommand() string {
```
Comment (jun-rh) [inline pkg/tnf/handlers/common/command.go:11]: this whole file can probably be removed. it was introduced solely for customized image id

Comment (javierpena) [reply pkg/tnf/handlers/common/command.go:11]: Good idea, I'll check it out.

File: pkg/tnf/handlers/readremotefile/readremotefile.go [hunk 2]
```diff
@@ -43,7 +42,7 @@ func NewReadRemoteFile(timeout time.Duration, nodeName, filePath string) *ReadRe
 		timeout: timeout,
 		result:  tnf.ERROR,
 		args: []string{
-			"echo", "\"cat /host" + filePath + "\"", "|", common.GetOcDebugCommand(), "-q", "node/" + nodeName,
+			"echo", "\"cat /host" + filePath + "\"", "|", "oc", "debug", "-q", "node/" + nodeName,
```
Comment (javierpena) [inline pkg/tnf/handlers/readremotefile/readremotefile.go:45]: Actually, I think this readremotefile handler could be completely removed, I can't find any place using it. However, let's do that post-3.2

Comment (jun-rh) [reply pkg/tnf/handlers/readremotefile/readremotefile.go:45]: Yes, we have quite a few handlers that should have been removed/deprecated.

**Review by jun-rh — APPROVED** [review]

**Pull request merged**